constants.gradle

```diff
@@ -1,10 +1,15 @@
-project.ext.firebase_version = '10.2.0'
-project.ext.support_library_version = '25.1.1'
+project.ext {
+    submodules = ['database', 'auth', 'storage']
+    group = 'com.firebaseui'
+    version = '1.1.1'
+    pomdesc = 'Firebase UI Android'
 
-project.ext.submodules = ['database', 'auth', 'storage']
-project.ext.group = 'com.firebaseui'
-project.ext.version = '1.1.1'
-project.ext.pomdesc = 'Firebase UI Android'
-project.ext.buildTools = '25.0.2'
-project.ext.compileSdk = 25
-project.ext.targetSdk = 25
+    compileSdk = 25
+    targetSdk = 25
+    minSdk = 14
+
+    buildTools = '25.0.2'
+
+    firebaseVersion = '10.2.0'
+    supportLibraryVersion = '25.1.1'
```

Comment:
I cleaned this up a little and made these names more consistent with `compileSdk`, `targetSdk`, etc.